### PR TITLE
Docker improvements (and some other minor stuff)

### PR DESCRIPTION
**Pull request by andreasEIQ**
Docker improvements:
- [x] don't use supervisor
- [x] use multi-stage builds to reduce image size
- [x] use python3 image

Other minor stuff:
- [x] get rid of duplicate readme files
- [x] fix badges/links in readme
- [x] update classifiers

### CODE REVIEW

**Comment by andreasEIQ** [comment]
Somehow tests are consistently failing on Python3.5 only :(

**Comment by andreasEIQ** [comment]
I'm able to reproduce by installing a python3.5 virtualenv and just running one of the failing tests over and over again. They fail like 20% of the time. Maybe `sqlite` reliability has improved in 3.6 and higher.

**Comment by alex-lion** [comment]
ping me if/when it's ready for review

**Comment by andreasEIQ** [comment]
See https://github.com/eclecticiq/OpenTAXII/pull/126 for the flaky tests

**Comment by andreasEIQ** [comment]
@alex-lion if you want, be my guest! :)

**Comment by alex-lion** [comment]
@andreasEIQ give me an hour finish my current activities and I'll be right back to it :)

File: Dockerfile [hunk 1]
```diff
@@ -1,42 +1,29 @@
-# Set the base image to Python
-FROM python:2.7.9
-MAINTAINER EclecticIQ <opentaxii@eclecticiq.com>
+FROM python:3.6-stretch AS build
+LABEL maintainer="EclecticIQ <opentaxii@eclecticiq.com>"
 
-# Create the working dir and set the working directory
-WORKDIR /
+RUN python3 -m venv /venv && /venv/bin/pip install -U pip setuptools
```
Comment (alex-lion) [inline Dockerfile]: You don't pin package versions, okay :). You know my opinion, but I don't insist.

File: Dockerfile [hunk 2]
```diff
@@ -1,42 +1,29 @@
...
 
-# Install dependencies
-COPY ./requirements.txt /requirements.txt
-RUN pip install -r requirements.txt \
-    && pip install supervisor \
-    && pip install gunicorn \
-    && pip install psycopg2 \
-    && rm -f /requirements.txt
+RUN /venv/bin/pip install gunicorn psycopg2-binary
```
Comment (alex-lion) [inline Dockerfile]: are these test requirements? Why they are not in `requirements-dev.txt`? (I'll approve anyway, it's not essential)

Comment (andreasEIQ) [reply Dockerfile]: They are production requirements for the docker image, but they are *not* requirements that should be installed if you do `pip install opentaxii`.

Maybe we should consider version-pinning these, though. And maybe we can put them in `requirements-docker.txt` for consistency?

File: Dockerfile (hunk 1 above)
Comment (andreasEIQ) [reply Dockerfile]: For `pip` and `setuptools` in particular, I'm of the opinion that our packages should *always* work with the latest versions, so I'm okay with not version-pinning them.

**Comment by alex-lion** [comment]
Cool!

**Pull request merged**